Discuss exception to access control (#16)
ietf-wg-keytrans/draft-arch: draft-ietf-keytrans-architecture.md

verified with no/minimal network requests by the receiving users.

In particular, credentials that can be verified with minimal network access are

> often required by applications that provide anonymous communication. These

Reviewer: maybe like this to avoid using `that + verb` twice in the same sentence?

```suggestion
often required by applications providing anonymous communication. These
```

the user obtains a valid search proof for the exact key and version stored at
that log entry.

When a user was previously allowed to lookup or change a label's value but no
longer is, KT prevents the user from learning whether or not the label's value
has changed since the user's access was revoked. Note however that in Contact

> Monitoring mode, users will still be permitted to perform monitoring to

Reviewer: ```suggestion
Monitoring mode, users SHOULD be permitted to perform monitoring to
```
